x-pack/plugins/security_solution/public/management/components/endpoint_responder/components/offline_callout.tsx

```diff
@@ -32,16 +33,20 @@ export const OfflineCallout = memo<OfflineCalloutProps>(({ agentType, endpointId
   const sentinelOneManualHostActionsEnabled = useIsExperimentalFeatureEnabled(
     'sentinelOneManualHostActionsEnabled'
   );
+  const crowdstrikeManualHostActionsEnabled = useIsExperimentalFeatureEnabled(
+    'responseActionsCrowdstrikeManualHostIsolationEnabled'
+  );
 
   const { data: endpointDetails } = useGetEndpointDetails(endpointId, {
     refetchInterval: 10000,
     enabled: isEndpointAgent,
   });
 
   const { data } = getAgentStatus([endpointId], agentType, {
-    enabled: sentinelOneManualHostActionsEnabled && isSentinelOneAgent,
+    enabled:
+      (sentinelOneManualHostActionsEnabled && isSentinelOneAgent) ||
+      (crowdstrikeManualHostActionsEnabled && isSentinelOneAgent),
   });
-
```

Comment:
Oh! I missed updating this component looks like. 😓  

`endpointDetails` should be fetched only if `agentStatusClientEnabled` is `false`. When it is `true` it should use `data` fetched from `getAgentStatus` to check for agent status.